src/Microsoft.IdentityModel.Tokens/Json/JsonSerializerPrimitives.cs

```diff
@@ -648,6 +648,33 @@ internal static string ReadString(ref Utf8JsonReader reader, string propertyName
             return reader.GetString();
         }
 
+        /// <summary>
+        /// This method is called to enable the 'sub' claim to deserialize it as a Number then returns it as a string.
```

Comment:
nit: If it is for sub claim only, consider rename it to ReadSubClaimAsString or don't mention the claim in summary

Reply:
I agree, I'll update the summary. Thanks Xiao!